src/modules/GameConstants.ts

```diff
@@ -26,7 +26,10 @@ export const TotalPokemonsPerRegion = [
     649, // 156 - Unova
     721, // 72 - Kalos
     809, // 88 - Alola
-    893, // 84 - Galar
+    890, // 81 - Galar
+    893, // 3 - Armor
+    898, // 5 - Crown
```

Comment:
```suggestion
    893, // 3 - Isle of Armor
    898, // 5 - Crown Tundra
```

Reply:
I was thinking the same, I'm just not sure how they will work as separate maps. My intention is to have them as post-game areas similar to the battle area in Sinnoh